src/install.c

```diff
@@ -1135,7 +1135,16 @@ gboolean do_install_bundle(RaucInstallArgs *args, GError **error) {
 	res = TRUE;
 
 umount:
-	umount_bundle(mountpoint, NULL);
+	{
+		/* Do not cover previous error but report if no other occurred, yet */
+		GError *iierror = NULL;
+		if (!umount_bundle(mountpoint, &iierror)) {
+			if (!ierror)
+				ierror = iierror;
```

Comment:
ierror is not used below this point. Did you want to propagate it?
This patch is also missing a SoB line.